core/templates/dev/head/domain/utilities/AutogeneratedAudioLanguageObjectFactory.ts

```diff
@@ -50,9 +50,9 @@ export class AutogeneratedAudioLanguageObjectFactory {
     return new AutogeneratedAudioLanguage(
       autogeneratedAudioLanguageDict.id,
       autogeneratedAudioLanguageDict.description,
-      autogeneratedAudioLanguageDict.exploration_language,
-      autogeneratedAudioLanguageDict.speech_synthesis_code,
-      autogeneratedAudioLanguageDict.speech_synthesis_code_mobile);
+      autogeneratedAudioLanguageDict.explorationLanguage,
```

Comment:
Remove the comment on lines 44-47. (That's what we want to solve through #7176, right?)

Reply:
@DubeySandeep, I made the changes you requested. But while trying to push the changes, a certain test seemed to fail. I have attached the screenshot below. This test doesn't seem to fail because of my changes. Also, @iamprayush is facing the same issue. Can you please see and tell if there is some mistake on my part. Thanks!
![Screenshot (118)](https://user-images.githubusercontent.com/44376010/73384694-c2bded80-42f1-11ea-95c9-1679fce6106b.png)

Reply:
This looks like an unrelated failure? Maybe you should merge with develop and try to run the tests...